portfolio/src/main/java/com/google/sps/utils/CommentDataStore.java

```diff
@@ -33,14 +32,17 @@ public void store(Comment comment) {
      * @return An Iterable.
      */
     @Override
-    public List<Comment> load() {
+    public List<Comment> load(int limit) {
 
         Query query = new Query(KIND).addSort(TIMESTAMP, Query.SortDirection.DESCENDING);
```

Comment:
Is it possible to make limit part of the querying?

Reply:
Fixed

Reply:
Fixed